src/lightgbm_R.cpp

```diff
@@ -9,15 +9,15 @@
 #include <LightGBM/utils/openmp_wrapper.h>
 #include <LightGBM/utils/text_reader.h>
 
+#include <R_ext/Rdynload.h>
+
```

Comment:
ah ok, thanks! I should have looked at `cpplint` manually in the logs when I did #2911

Reply:
I hope soon we will not have to do this.

Reply:
+1 I agree